atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CellValuePutter.java

```diff
@@ -101,6 +115,9 @@ public Void apply(CassandraClient client) throws Exception {
                             for (Map.Entry<Cell, Value> e : partition) {
                                 Cell cell = e.getKey();
                                 Column col = CassandraKeyValueServices.createColumn(cell, e.getValue());
+                                if (overwriteTimestamps) {
```

Comment:
This is wasteful as you end up calling `createColumn` twice - this matters more if the value is large, as you'd extract it twice. It's easy to avoid:

```
Column col = overwriteTimestamps
        ? CassandraKeyValueServices.createColumnAndOverwriteTimestamp(cell, e.getValue())
        : CassandraKeyValueServices.createColumn(cell, e.getValue());
```